docs/docs/chitchat-faqs.mdx

```diff
@@ -26,65 +26,70 @@ In this section we're going to cover the following topics:
 
 We're going to use content from [Sara](https://github.com/RasaHQ/rasa-demo), the Rasa
 assistant that, amongst other things, helps the user get started with the Rasa products.
-You should first install Rasa using the [Step-by-step Installation Guide](./installation.mdx#step-by-step-installation-guide)
-and then [prototype an assistant](./prototype-an-assistant.mdx)
+You should [first install Rasa](installation.mdx)
+and then [prototype an assistant](prototype-an-assistant.mdx)
 to make sure you know the basics.
 
-To prepare for this tutorial, we're going to create a new directory and start a
-new Rasa project.
+To prepare for this tutorial, we're going to start a new Rasa project:
 
 ```bash
-mkdir rasa-assistant
 rasa init
 ```
 
-Let's remove the default content from this bot, so that the `data/nlu.md`, `data/stories.md`
+Let's remove the default content from this bot, so that the `data/nlu.yml`, `data/stories.yml`
 and `domain.yml` files are empty.
 
 <a aria-hidden="true" tabIndex="-1" className="anchor enhancedAnchor" id="respond-with-memoization-policy"></a>
 
 ## Memoization Policy
 
 The MemoizationPolicy remembers examples from training stories for up to a `max_history`
-of turns. The number of “turns” includes messages the user sent, and actions the
-assistant performed. For the purpose of a simple, context-less FAQ bot, we only need
-to pay attention to the last message the user sent, and therefore we'll set that to `1`.
+of turns. One “turn” includes the message sent by the user and any actions the
+assistant performed before waiting for the next message. For the purpose of a simple,
+context-less FAQ bot, we only need to pay attention to the last message the user sent,
+and therefore we'll set that to `1`.
 
-You can do this by editing your `config.yml` file as follows (you can remove `TEDPolicy` for now):
+You can do this by editing your configuration file as follows
+(you can remove `TEDPolicy` for now):
 
-```yaml
+```yaml title="config.yml"
 policies:
 - name: MemoizationPolicy
   max_history: 1
 - name: MappingPolicy
 ```
 
-:::note
-The MappingPolicy is there because it handles the logic of the `/restart` intent,
+:::note MappingPolicy
```

Comment:
should probably be part of different PR: they should use the RulePolicy instead of the MappingPolicy

Reply:
yes, but that is separate